test/metabase/automagic_dashboards/rules_test.clj

```diff
@@ -0,0 +1,45 @@
+(ns metabase.automagic-dashboards.rules-test
+  (:require [expectations :refer :all]
+            [metabase.automagic-dashboards.rules :refer :all :as rules]))
+
+(expect
+  [nil
+   [nil]
+   [42]
+   [42]]
+  (map #'rules/ensure-seq [nil [nil] 42 [42]]))
+
+(expect
+  [true
+   false]
+  (map ga-dimension? ["ga:foo" "foo"]))
```

Comment:
I'm not really sure what's going on in this namespace -- what's the reasoning behind trying to combine a bunch of tests that anywhere else in the codebase would be separate into one single test? I just don't understand why